app/src/main/java/com/duckduckgo/app/privacy/ui/WebsitesAdapter.kt

```diff
@@ -107,7 +107,7 @@ class WebsitesAdapter(
                 WebsiteViewHolder.SimpleViewHolder(binding.root)
             }
             DIVIDER_TYPE -> {
-                WebsiteViewHolder.SimpleViewHolder(SectionDivider(parent.context))
+                WebsiteViewHolder.SimpleViewHolder(HorizontalDivider(parent.context))
```

Comment:
This one too:

Before
-------
![before2](https://user-images.githubusercontent.com/3471025/192835837-d965c71b-4039-4010-9130-0f8c291e5754.png)

After
------
![after2](https://user-images.githubusercontent.com/3471025/192835899-26561329-c047-411e-a845-50078bcf932a.png)

Reply:
fixed